tests/playwright/shiny/components/express_navs/modules.py

```diff
@@ -0,0 +1,70 @@
+from shiny import reactive
+from shiny.express import ui, module
+
+
+@module
+def add_to_counter_content(
+    input, output, session, _on_click, starting_value, label="Add to Counter"
+):
+    count = reactive.value(starting_value)
+    ui.input_action_button("add", label)
+
+    @reactive.effect
+    @reactive.event(input.add)
+    def increment_button():
+        _on_click()
```

Comment:
This callback will change the state of it's parent session, which I'd generally consider an anti-pattern.

If reactive information needs to be shared (which, btw, I'm not sure we need here?), then the best way to do that is to pass reactive the values themselves https://shiny.posit.co/r/articles/improve/communicate-bet-modules/